### DISCUSSION TITLE
How to add a group of View buttons

### DISCUSSION THREAD

**Discussion by lohhb** [Budibase/budibase#14463] [Help] [answered]
How can I add a group of View button?
I tried to add but was not able to add as a group.
Refer sample from Budibase template:
![View button](https://github.com/user-attachments/assets/145355f2-24af-4137-bcea-e4c455839c12)

**Comment by lohhb** [comment]
I figured out!

**Comment by mikesealey** [comment, marked as answer]
For anyone else wondering: At the bottom of the settings for a Table component you can add buttons. Clicking the little cog icon on each button will open up the options menu and let you set up that button.
![image](https://github.com/user-attachments/assets/00e6b5fe-bbcf-4f12-b35a-49fbed4a307a)

Worth noting: The table displayed in @lohhb 's original post is the deprecated Table Block component.
